src/tram/tram/management/commands/pipeline.py

```diff
@@ -50,11 +51,17 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         subcommand = options["subcommand"]
 
+        user, created = User.objects.get_or_create(username="pipeline (manual)")
+        if created:
+            self.stdout.write(
```

Comment:
One thing I've noticed throughout this project is that we use print() or stdout.write() instead of logging. We should open an issue to set up logging.

Reply:
resolved on #153